e2e/tests/assorted/09-joinfromdirectory.spec.js

```diff
@@ -0,0 +1,45 @@
+const {
+	device, expect, element, by, waitFor
+} = require('detox');
+const data = require('../../data');
+const { navigateToLogin, login, tapBack } = require('../../helpers/app');
+
+const testuser = data.users.regular
+
+async function navigateToRoom(search) {
+	await element(by.id('federation-view-search')).replaceText(search);	
+	await waitFor(element(by.id(`federation-view-item-${ search }`))).toBeVisible().withTimeout(10000);
+	await expect(element(by.id(`federation-view-item-${ search }`))).toExist();
+	await element(by.id(`federation-view-item-${ search }`)).tap();
```

Comment:
Same issue as others.
This function is supposed to `navigate to room`, but last action is just a tap on an item.
You're not testing if the app really navigated to the room.